static/src/js/components/PermissionForm/PermissionForm.jsx

```diff
@@ -37,6 +37,7 @@ import { cloneDeep } from '@apollo/client/utilities'
 import {
   GET_COLLECTION_FOR_PERMISSION_FORM
 } from '@/js/operations/queries/getCollectionForPermissionForm'
+import useAvailableProviders from '@/js/hooks/useAvailableProviders'
```

Comment:
When you come across files that are using a mix of `@` and `../` imports, please update them to use `@` -- this was introduced later in the effort and it didn't make sense to transition every file. So, convert the other imports to use `@` and alphabetize the imports please.

Reply:
done